talent/hiring_process.md

```diff
@@ -0,0 +1,77 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+### Job Description
+
+- Job Description is written by the lead engineers with the Talent's Manager support. 
+
+- For every meeting between the lead and the Talent Manager there will be a PR in the [minutes](https://github.com/src-d/minutes) repo and the job descriptions will be uploaded at the private repository https://github.com/src-d/company.
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
+
+- The Talent Manager posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
+
+### Screening
+
+- If the developer is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
+
+- If the developer is approached by us from a specific source (known from meetups, having worked together in the past,  etc.), they should move forward to the next step as it means that the leads have already shown interest.
+
+### First Skype/Zoom/Phone Interview
+
+- A quick Skype/Phone/Zoom call should be available to every developer who is interested in the roles*. It is a general discussion to know more about the background and experience of the developer. And to share as much as possible from our side about the team & company. 
+
+\* There a noteable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call. 
+
+- Lead engineers are responsible to make the calls as it's a motivating factor for the developers. Leads will also be able to assess early in the process if the interview process should continue or not. 
+
+### Coding Challenges 
+
+- Every developer who passes the first Skype interview, should complete the coding challege. 
+
+- In case of the Machine Learning team, there is an additional ML task necessary to be solved.
+
+- Coding Challeges are being sent to the developers by the Talent Manager and the results are being reviewed by the lead engineers. 
+
```

Comment:
We had some "second chance" exceptions in the past. When the guy was super disappointed and showed his motivation and cried out loud, we sent the second coding challenge.

Reply:
As this will be posted in the guide which is public, if the candidates read it they might all request a second chance. Also even if we do give second chance (for very specific reasons), it doesn't seem a consistent approach and not sure if it's a good idea to be in this guide.

Reply:
Right. Stupid transparency